monica.json

```diff
@@ -6,31 +6,31 @@
     "pkgs": [
         "git-lite",
         "pwgen-2.08,2",
-        "apache24-2.4.46",
+        "apache24-2.4.46_1",
         "mysql57-server",
-        "redis-5.0.9",
+        "redis5-5.0.10",
         "mod_php74",
-        "php74-composer-1.10.15",
-        "php74-bcmath-7.4.12",
-        "php74-curl-7.4.12",
-        "php74-dom-7.4.12",
-        "php74-gd-7.4.12",
-        "php74-gmp-7.4.12",
-        "php74-iconv-7.4.12",
-        "php74-intl-7.4.12_1",
-        "php74-json-7.4.12",
-        "php74-mbstring-7.4.12",
-        "php74-mysqli-7.4.12",
-        "php74-opcache-7.4.12",
-        "php74-pdo_mysql-7.4.12",
+        "php74-composer-1.10.20",
+        "php74-bcmath-7.4.15",
+        "php74-curl-7.4.15",
+        "php74-dom-7.4.15",
+        "php74-gd-7.4.15",
+        "php74-gmp-7.4.15",
+        "php74-iconv-7.4.15",
+        "php74-intl-7.4.15",
+        "php74-json-7.4.15",
+        "php74-mbstring-7.4.15",
+        "php74-mysqli-7.4.15",
+        "php74-opcache-7.4.15",
+        "php74-pdo_mysql-7.4.15",
```

Comment:
This is likely to break very soon. Do we have to use specific version?

Reply:
Good point, I added this as a quickfix for the already broken versions but also realized this will break after these packages are upgraded again. Can fix a new PR removing the versions.

Reply:
Would this be better: https://github.com/ix-plugin-hub/iocage-plugin-index/pull/183 ?

Reply:
Yup, thanks!